airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeS3StagingSqlOperations.java

```diff
@@ -86,16 +86,18 @@ public void createStageIfNotExists(final JdbcDatabase database, final String sta
   }
 
   @Override
-  public void copyIntoRawTableFromStage(final JdbcDatabase database,
+  public void copyIntoTargetTableFromStage(final JdbcDatabase database,
```

Comment:
Now that we are not using temp tables anymore, the only table this connector interacts with is the `_raw` table, which we are addressing as `targetTable` in parameter and method names. Would it make sense to drop the `target` and `raw` words from parameter and method names altogether and just call it `table`:
 - `tableName`
 - `copyIntoTableFromStage`